INSTALL

```diff
@@ -41,9 +41,10 @@ a development ('-dev' or '-devel') package separate from the main library.
 Disabling libbzip2 and liblzma will make some CRAM files unreadable, so
 is not recommended.
 
-Using libcurl provides HTSlib with better network protocol support, for
-example it enables the use of https:// URLs.  It is also required if
-direct access to Amazon S3 or Google Cloud Storage is enabled.
+Using libcurl provides HTSlib withnetwork protocol support, for
+example it enables the use of ftp and https:// URLs.  It is also
+required if direct access to Amazon S3 or Google Cloud Storage is
+enabled.
```

Comment:
Fix typo (missing space), and suggested spelling out all the important URL schemes:

```suggestion
Using libcurl provides HTSlib with network protocol support, for example it
enables the use of ftp://, http://, and https:// URLs.  It is also required
if direct access to Amazon S3 or Google Cloud Storage is enabled.
```

Reply:
Odd.  I checked the original and the missing space is actually an entire missing " better " word.  Must have been some odd finger slippage.

Anyway thanks for the updates.  I'll amend.

Reply:
Removing “better“ here was clearly the right thing to do :smile: Now I'm off to get my third coffee of the morning, to fend off my own befuddlement…